docs/docs/index.md

```diff
@@ -82,7 +83,7 @@ In the last case, it may be advantageous to instead use the explicit version of
 q = Queue('low', connection=redis_conn)
 q.enqueue_call(func=count_words_at_url,
                args=('http://nvie.com',),
-               timeout=30)
+               job_timeout=30)
```

Comment:
This appears to be incorrect? It doesn't seem like the call signature for `enqueue_call()` was changed?